apps/system/js/app_window.js

```diff
@@ -926,6 +923,13 @@
         this[componentName] = null;
       }
 
+      for (var propertyName in this.constructor.SUB_MODULES) {
+        if (this[propertyName] && this[propertyName].stop) {
+          this[propertyName].stop();
+        }
+        this[propertyName] = null;
+      }
+
```

Comment:
Do we still need to add this stopping logic in `AppWindow` now that we moved `BrowserContextMenu` into `AppWindowManager`? Perhaps this is just for consistency?